src/main/java/org/spongepowered/common/data/DataRegistrar.java

```diff
@@ -628,6 +628,11 @@ public static void setupSerialization(Game game) {
         dataManager.registerValueProcessor(Keys.DYE_COLOR, new SheepDyeColorValueProcessor());
         dataManager.registerValueProcessor(Keys.DYE_COLOR, new ItemDyeColorValueProcessor());
 
+        final BreedableDataDualProcessor breedableDataDualProcessor = new BreedableDataDualProcessor();
+        dataManager.registerDataProcessorAndImpl(BreedableData.class, SpongeBreedableData.class, ImmutableBreedableData.class,
```

Comment:
Now you just call `registerDualProcessor` and don't need `registerValueProcessor` anymore